AEPMessaging/Sources/Messaging+PublicAPI.swift

```diff
@@ -121,7 +155,7 @@ import UserNotifications
                           source: EventSource.requestContent,
                           data: eventData)
 
-        MobileCore.dispatch(event: event) { responseEvent in
+        MobileCore.dispatch(event: event, timeout: 1) { responseEvent in
```

Comment:
thoughts on exposing this timeout somehow?

Reply:
Yes, we will need to do it at some point. Customers have been asking for such a control across the SDK APIs. Probably qualifies for a SDK guild discussion.